css/images.css

```diff
@@ -0,0 +1,18 @@
+.image-container {
+  width: 100%;
+}
+
+.image {
+  width: 100%;
+  height: auto;
+}
+
+.image.round {
+  border-radius: 50%;
+}
+
+@media screen and (min-width: 768px) {
+  .image-container {
+    width: 50%;
```

Comment:
This media query is not needed in that case. Try it out! :)

Reply:
Solved